DEVELOPMENT.md

```diff
@@ -1,5 +1,14 @@
 # Development Guidelines
 
+## Getting Started
+
+You would need a local clone of the `dandi-archive` repository to develop on it.
+
+1. Run `git clone https://github.com/dandi/dandi-archive`
+2. Run `cd dandi-archive`
+3. Also make sure you are not already running a local PostgreSQL server on port 5432 (recommended), or
```

Comment:
I dont think this makes sense here-- we arent running Postgress until `docker compose up` for quickstart, or ``docker compose -f ./docker-compose.yml up` for advanced setup.

Reply:
Nevermind, I misread. 

```suggestion
3. Make sure your PostgreSQL port (5432) is available (recommended), or
```